internal/commands/vorpal-engine_test.go

```diff
@@ -0,0 +1,122 @@
+package commands
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/commands/util/printer"
+	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/spf13/cobra"
+)
+
+func Test_executeVorpalScan(t *testing.T) {
```

Comment:
Can the name be consistent? (Camel case? pascal case?)

Reply:
It kept it lowercased because the method used to be private. Changed it accordingly now.